website/versioned_docs/version-v1.x/testing.md

```diff
@@ -169,7 +169,7 @@ You can run them with:
 pnpm run test
 ```
 
-## Pre-Compiled Artifacts for testing
+## Pre-Compiled Artifacts for testing
```

Comment:
Not sure why this fixes it but previously wasn't working:
![Image 2024-01-13 at 13 39 47](https://github.com/privacy-scaling-explorations/maci/assets/8097623/1002bf44-1d3c-48aa-9c50-b86374025f83)